web/src/beta/components/fields/PropertyFields/index.tsx

```diff
@@ -15,119 +18,174 @@ import useHooks from "./hooks";
 
 type Props = {
   propertyId: string;
-  item?: Item;
+  item: Item;
   currentCamera?: Camera;
   onFlyTo?: FlyTo;
 };
 
 const PropertyFields: React.FC<Props> = ({ propertyId, item, currentCamera, onFlyTo }) => {
-  const { handlePropertyValueUpdate } = useHooks();
+  const {
+    handlePropertyValueUpdate,
+    handleAddPropertyItem,
+    handleRemovePropertyItem,
+    handleMovePropertyItem,
+  } = useHooks(propertyId, item.schemaGroup);
+
+  // Just for the ListItem Property
+  const isList = item && "items" in item;
+
+  // TODO: Only applies to list, should be refactored
+  const [selected, setSelected] = useState<string | undefined>();
+
+  // TODO: Only applies to list, should be refactored
+  const propertyListItems: Array<{ id: string; value: string }> = useMemo(
+    () =>
+      isList
+        ? item.items
+            .filter(i => "id" in i)
+            .map(i => {
+              const representativeField = item?.representativeField
+                ? i.fields.find(f => f.id === item.representativeField)
+                : undefined;
+              const nameSchemaField = item?.schemaFields?.find(
+                sf => sf.id === item.representativeField,
+              );
+
+              const value = representativeField?.value || nameSchemaField?.defaultValue;
+
+              const choice = nameSchemaField?.choices
+                ? nameSchemaField?.choices?.find(c => c.key === value)?.label
+                : undefined;
+
+              const title = choice || value;
+
+              return {
+                id: i.id,
+                value: title as string,
+              };
+            })
+        : [],
+    [isList, item],
+  );
+
+  const showFields = useMemo(() => {
+    return isList ? (selected ? item.items.find(({ id }) => id == selected) : false) : true;
```

Comment:
90% of the time triple equals is desired 👍🏻 
So should be `id === selected`

Reply:
Done Done.